app/views/admin/confirm_metadata.html.erb

```diff
@@ -33,14 +33,6 @@
       previous_schema: @current_format.finder_schema.schema,
     } %>
 
-    <%= render "govuk_publishing_components/components/details", {
-      title: "Proposed JSON",
```

Comment:
Given the detail is hidden away, I wonder if it's at all useful to keep this in? Up to you.

Reply:
I can see how it might be useful for us, but it also breaks the layout when it's opened.
I'm keen to remove it for now and if we find that we have a genuine need for it add a properly thought out solution.